curator-client/src/main/java/org/apache/curator/HandleHolder.java

```diff
@@ -73,15 +73,15 @@ String getNewConnectionString()
         return ((helperConnectionString != null) && !ensembleProvider.getConnectionString().equals(helperConnectionString)) ? helperConnectionString : null;
     }
 
-    void closeAndClear() throws Exception
+    void closeAndClear(int timeout) throws Exception
```

Comment:
Please rename argument to `waitForShutdownTimeoutMs`

Reply:
done